images/win/windows2016.json

```diff
@@ -137,7 +136,6 @@
         {
             "type": "powershell",
             "inline": [
-                "setx ImageVersion {{user `image_version` }} /m",
```

Comment:
We mustn't remove this environment variable. It is set intentionally so customers can read it and determine image version that they use

Reply:
I understand what the difference is now between the `ImageVersion` and `IMAGE_VERSION` env vars.
It would be good to have had that documented, however, I bet you get that a lot :)

I have updated the PR. As @al-cheb suggested originally I have moved the `setx` into the initialize script. However, due to the amount of change, there was to remove `IMAGE_VERSION` I have decided against that in favour of continuing using this temp format as the pattern that is used throughout the packer file and PS files.